src/features/game/lib/gameMachine.ts

```diff
@@ -323,48 +325,52 @@ const playingEventHandler = (eventName: string) => {
 
           return !valid;
         },
-        actions: assign((context: Context, event: PlayingEvent) => {
-          const { maxedItem } = checkProgress({
-            state: context.state as GameState,
-            action: event,
-            farmId: context.farmId,
-          });
+        actions: assign(
```

Comment:
The `gameMachine` updates have caused bugs in the effects again. Unable to craft, fish, airdrop players etc. 👀 

The game freezes or errors when trying to perform these actions